### PR TITLE
Add perlgov vote administrator guide to Porting

### PR DESCRIPTION
**Pull request by xdg**
This commit adds a guide to perlgov vote administrators for how to run
Perl Steering Council elections.

### CODE REVIEW

File: Porting/vote_admin_guide.pod [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+You'll need to complete these steps.
+
+=head3 Get list of active Core Team member emails
+
+Ask Rik Signes for an up-to-date list of active Core Team member emails from
+the TopicBox mailing list.
+
+B<TODO>: Replace this step with a machine-readable list of email addresses in
+C<Porting>.
```
Comment (jkeenan) [inline Porting/vote_admin_guide.pod]: On the perl-core mailing list, there has been a discussion of the different email addresses which various committers use for different purposes.  Do we need to resolve that before we can address the B<TODO>?

**Review by jkeenan — COMMENTED** [review]
@xdg, can you give this p.r. a once-over to see if anything needs revising in light of the poll concluding soon.

**Comment by jkeenan** [comment]
Note: Whoever ultimately merges this may want to use `Porting/add-pod-file` to insert the new document.

**Comment by xdg** [comment]
I have no other additions to make.  I think the TODO should remain until the new email list is added, and this document can be updated as part of that commit.

File: Porting/vote_admin_guide.pod (hunk 1 above)
Comment (rjbs) [reply Porting/vote_admin_guide.pod]: I believe #18941 resolves this.

File: Porting/vote_admin_guide.pod [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+time to close the nominations and begin the voting.
+
+If someone outside the Core Team is nominated, contact them to confirm that
+they accept the nomination.  If they do, ask the Steering Council to add
+them as a moderated member of the Perl Core list for the duration of the
+election and invite them to offer a statement on their candidacy to the
+list.
+
+Remind inactive Core Team members that they have until the end of
+nominations to reactivate themselves if they wish to vote.
```
Comment (rjbs) [inline Porting/vote_admin_guide.pod]: perlgov says this:
> Core Team members who have stopped contributing are encouraged to declare themselves "inactive". Inactive members do not nominate or vote.  Inactive members may declare themselves active at any time, except when a vote has been proposed and is not concluded.  Eligibility to nominate or vote will be determined by the Vote Administrator.

When is a vote "proposed" in this case?  I would have thought that when nominations begin, the vote is proposed, and an inactive member must wait it out.

File: Porting/vote_admin_guide.pod [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+need to save it to come back to what you were doing before.
+
+=head2 Preliminary work
+
+You'll need to complete these steps.
+
+=head3 Get list of active Core Team member emails
+
+Ask Rik Signes for an up-to-date list of active Core Team member emails from
+the TopicBox mailing list.
```
Comment (rjbs) [inline Porting/vote_admin_guide.pod]:
```suggestion
the Topicbox mailing list.
```

(Corrects capitalization.)

File: Porting/vote_admin_guide.pod [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+Inform the group!
+
+=head2 After an election
+
+In addition to notifying the Perl Core mailing list of the results, open a
+pull request to update perlgov.pod with the new Steering Council members.
+If a non-Core-Team candidate won, also add them to the Core Team list.
+
+Ask the outgoing Steering Council to change the Perl Core mailing list
+administrators to match the incoming Steering Council.
```
Comment (rjbs) [inline Porting/vote_admin_guide.pod]:
```suggestion
Ask the outgoing Steering Council to:

=over

=item *

edit the Perl Core mailing list admins to match the incoming Steering Council

=item *

update the GitHub "steering" team to match incoming Steering Council

=item *
 
request that the Perl NOC update the perl-security list to include all new council members

=back
```

**Review by rjbs — CHANGES_REQUESTED** [review]

**Comment by rjbs** [comment]
@xdg This looks good, thanks!  I made a few suggestions and asked an important question.

File: Porting/vote_admin_guide.pod (hunk 2 above)
Comment (xdg) [reply Porting/vote_admin_guide.pod]: That's a good point and we've not been doing that.  I'll update this.

File: Porting/vote_admin_guide.pod (hunk 3 above)
Comment (xdg) [reply Porting/vote_admin_guide.pod]: Moot, since it's replaced with `core-team.json`.

File: Porting/vote_admin_guide.pod (hunk 1 above)
Comment (xdg) [reply Porting/vote_admin_guide.pod]: I will include a reference to `core-team.json` and optimistically assume that both PRs will eventually merge.

File: Porting/vote_admin_guide.pod (hunk 4 above)
Comment (xdg) [reply Porting/vote_admin_guide.pod]: Edited and included in a separate commit.

**Review by rjbs — APPROVED** [review]

**Comment by xdg** [comment]
Merged as bfb2c8dfe0

**Pull request closed without merging**